assignment/scripts/function-practice.js

```diff
@@ -49,39 +63,71 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
-
+function getLast(array) {
+  if (array.length === 0){
+    return 'undefined';
```

Comment:
Not quite, though I understand the confusion. The prompt uses backticks, which in JavaScript create a (template) string. However, a common convention when writing documentation is to use backticks to indicate _code_. So the prompt in this case wanted you to return the _value_ `undefined`, not the _string_ "undefined".